### PR TITLE
[ADVAPP-1038]: Purge experimental reporting from the system

### PR DESCRIPTION
**Pull request by danharrin**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-1038

### Technical Description

Removed feature, adjusted settings table to remove references, and registered legacy migrations directory so they are still run.

### Any deployment steps required?

No

### Are any Feature Flags and/or Data Migrations that can eventually be removed Added?

No, data migration needs to remain since the settings are created in previous migrations

_______________________________________________

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -93,6 +93,8 @@ public function register(): void
                 return $app->make(CacheFactory::class);
             });
         });
+
+        $this->loadMigrationsFrom(database_path('migrations/Legacy'));
```
Comment (Orrison) [inline app/Providers/AppServiceProvider.php:97]: Were Legacy migrations not running before?

File: database/migrations/Legacy/2024_11_29_133232_data_drop_report_assistant_settings.php [hunk 2]
```diff
@@ -30,6 +32,29 @@
...
---}}
+*/
+
+use Database\Migrations\Concerns\CanModifySettings;
+use Spatie\LaravelSettings\Migrations\SettingsMigration;
+
+return new class () extends SettingsMigration {
+    use CanModifySettings;
+
+    public function up(): void
```
Comment (Orrison) [inline database/migrations/Legacy/2024_11_29_133232_data_drop_report_assistant_settings.php:43]: Should this have a down to reverse?

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app/Providers/AppServiceProvider.php (hunk 1 above)
Comment (danharrin) [reply app/Providers/AppServiceProvider.php:97]: As far as I could tell, no

File: database/migrations/Legacy/2024_11_29_133232_data_drop_report_assistant_settings.php (hunk 2 above)
Comment (danharrin) [reply database/migrations/Legacy/2024_11_29_133232_data_drop_report_assistant_settings.php:43]: Done

**Review by Orrison — APPROVED** [review]

**Pull request merged**